### ISSUE TITLE
[bug] phero-client is not installed

### ISSUE THREAD

**Issue by draylegend** [phero-hq/phero#87]
### Given

Following the `getting started` guide, got this error:

```js
phero 
◥
phero-client @ ./client ◢ Loading...
phero-server @ ./api    ◢ Loading...
d:\dev\progs\node\node_modules\phero\node_modules\yoga-layout-prebuilt\yoga-layout\build\Release\nbind.js:53
        throw ex;
        ^

Error: phero-client is not installed in D:\dev\web\tests\phero\client
    at ChildProcess.<anonymous> (d:\dev\progs\node\node_modules\phero\dist\process.js:37:27)
    at ChildProcess.emit (node:events:520:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:289:12)
    at onErrorNT (node:internal/child_process:478:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

### Expected

Server should start

**Comment by Jpunt** [comment]
Thanks for reaching out 👍 Before we can find out what’s going on, can you tell me a little bit about your environment?

**Comment by draylegend** [comment]
@Jpunt what info exactly do you need?

**Comment by Jpunt** [comment]
Looking at the path I’m guessing it’s windows? What version of the phero-packages are you using? What frontend-framework is it? ☺️

**Comment by draylegend** [comment]
win10 x64
node v16.14.0
yarn 1.22.18
typescript 4.9.3
@phero/server 0.9.13
@phero/client 0.9.13

Phero only. No FE framework

**Comment by Jpunt** [comment]
Thanks, clear, we’ll get into it asap!

**Comment by the-unfactoring-guru** [comment]
Yeah I'm having the same problem:
O.S: Windows 11
Node: 18.12.1
Typescript: 4.9.4
Phero client: 0.9.13
Phero server: 0.9.19

**Comment by Jpunt** [comment]
This should be fixed in #93. I'll close this for now, but feel free to open a new issue if you've got any problems.
